.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
are we commiting IDE specific settings?

Reply:
Shoot. That was supposed to be ignored. No, this should get removed. Done.